resources/assets/js/photoAlbum.js

```diff
@@ -0,0 +1,239 @@
+
+var PhotoSwipeItems = []; // Contains the photos used by photoswipe.
+
+try {
+    var currentPhotoAlbum = photoAlbum.id;
+    for (var photo in photos.data) {
+        PhotoSwipeItems.push({ src: photos.data[photo].link, w: photos.data[photo].width, h: photos.data[photo].height });
+    }
+    $("p#albumDesciption").text($("p#albumDesciption").text().replace(/\n/g, "<br>"));
+}
+catch(err) {
+    $("p#albumDesciption").text($("p#albumDesciption").text().replace(/\n/g, "<br>"));
```

Comment:
Ik zou deze line uit de try en catch blocks halen, en in een finally block zetten. Het is misschien ook wel beter om iets van een alert te geven als er een error is.

Reply:
Fixed in commit 2be7866